import-service/src/handlers/importFileParser/handler.ts

```diff
@@ -0,0 +1,67 @@
+import { S3Event, S3EventRecord } from "aws-lambda";
+import { S3Client, GetObjectCommand, CopyObjectCommand, DeleteObjectCommand } from "@aws-sdk/client-s3";
+import csv from "csv-parser";
+import { Stream, Writable } from "stream";
+import util from "util";
+
+const pipelineAsync = util.promisify(Stream.pipeline);
+const s3 = new S3Client();
+const handleRecord = async (record: S3EventRecord) => {
```

Comment:
empty space please between function and variable

Reply:
done